run_server.sh

```diff
@@ -1,7 +1,9 @@
 #!/bin/sh
 echo "${@}"
-/opt/spire/tornjak-agent -c $2 http &
-/opt/spire/tornjak-agent -c /run/spire/config/server.conf http --tls --cert sample-keys/tls.pem --key sample-keys/key.pem  --listen-addr :20000 &
-/opt/spire/tornjak-agent -c /run/spire/config/server.conf http --mtls --cert sample-keys/tls.pem --key sample-keys/key.pem  --mtls-ca sample-keys/rootCA.pem --listen-addr :30000 &
+echo $1
+echo $2
+/opt/spire/tornjak-agent "${@}" --json-web-key-service http://host.minikube.internal:8080/realms/tornjak/protocol/openid-connect/certs http &
```

Comment:
I know this is just a default value to get things going, but I would suggest creating a local variable that holds the default value. For example:

```
CERTS_SERVER=${CERTS_SERVER:-"http://host.minikube.internal:8080/realms/tornjak/protocol/openid-connect/certs"}
# it takes the value of the env. variable with the same name, or it uses the default value
# so, later you can call it: 
/opt/spire/tornjak-agent "${@}" --json-web-key-service ${CERTS_SERVER} http &
```